core/electron-manager/src/backend/ElectronHost.ts

```diff
@@ -128,6 +127,8 @@ export class ElectronHost {
       // eslint-disable-next-line no-console
       // console.warn(`WARNING: Frontend requested "${requestedUrl}", but ${assetPath} does not exist`);
     }
+    if (!assetPath.startsWith(this.webResourcesPath))
+      throw new Error("Access outside that path is prohibited");

```

Comment:
"that path" is ambiguous - how does:

```suggestion
      throw new Error(`Access to files outside installation directory (${this.webResourcesPath}) is prohibited`);
```

sound?